### DISCUSSION TITLE
Autoresolvers did not work for me when field has a number in its name

### DISCUSSION THREAD

**Discussion by egor83** [mirumee/ariadne#632] [Q&A] [answered]
I had the following model (using Ariadne and Flask/SQLAlchemy):
```
class Company(db.Model):
    ...
    address1 = db.Column(db.String(255), nullable=False, server_default="")
    address2 = db.Column(db.String(255), nullable=True, server_default="")
    ...
```
and the following type in GraphQL:
```
type Company {
    ...
    address1: String
    address2: String
    ...
}
```

All other autoresolved fields worked fine, but `address1/address2` returned `null` (although they were not `null` in the DB, and I checked that the function called by resolver fetched actual non-null values).
I worked around it by writing explicit resolvers for these two fields, but still wanted to give you a heads-up - is it an Ariadne issue, or should I reach out to core GraphQL devs?

**Comment by rafalp** [comment, marked as answer]
This is by design. Automatic resolver maps `address1` in schema to `address_1` in python.
